clang/unittests/Format/FormatTest.cpp

```diff
@@ -12518,6 +12521,10 @@ TEST_F(FormatTest, UnderstandsUsesOfStarAndAmp) {
   verifyIndependentOfContext("MACRO(A *_Nonnull a);");
   verifyIndependentOfContext("MACRO(A *_Nullable a);");
   verifyIndependentOfContext("MACRO(A *_Null_unspecified a);");
+  verifyIndependentOfContext("MACRO(A *absl_nonnull a);", getGoogleStyle());
+  verifyIndependentOfContext("MACRO(A *absl_nullable a);", getGoogleStyle());
+  verifyIndependentOfContext("MACRO(A *absl_nullability_unknown a);",
+                             getGoogleStyle());
```

Comment:
```suggestion

  Style = getGoogleStyle();
  verifyIndependentOfContext("MACRO(A *absl_nonnull a);", Style);
  verifyIndependentOfContext("MACRO(A *absl_nullable a);", Style);
  verifyIndependentOfContext("MACRO(A *absl_nullability_unknown a);", Style);

```
Something like that.

Reply:
SG done!